cedar-lean/Protobuf/HardCodeTest.lean

```diff
@@ -0,0 +1,90 @@
+/-
```

Comment:
Can we remove this file or maybe move it to UnitTest?

Reply:
Removed.  Will re-add in the future if needed for other parts of #400.  Possibly in UnitTest instead of here.